src/components/reusable/table/Table.stories.ts

```diff
@@ -114,7 +114,7 @@ const tableRenderer = (args: any, updateArgs: any, title: string) => {
             <kyn-th>Col 4</kyn-th>
             <kyn-th>Col 6</kyn-th>
             <kyn-th>Col 7</kyn-th>
-            <kyn-th .align=${'center'}></kyn-th>
+            <kyn-th .align=${'center'} .visiblyHidden=${true}>Info Icon</kyn-th>
```

Comment:
To be true, you can just add `visiblyHidden`. It doesn't need the full binding treatment, `.visiblyHidden=${true}`, unless it's dynamic.